packages/app/src/app/overmind/namespaces/live/liveMessageOperators.ts

```diff
@@ -10,21 +9,23 @@ import { Operator } from 'app/overmind';
 import { camelizeKeys } from 'humps';
 import { NotificationStatus } from '@codesandbox/notifications/lib/state';
 
-export const onJoin: Operator<LiveMessage<LiveMessageEvent.JOIN>> = mutate(
-  ({ effects, state }) => {
-    effects.notificationToast.success(
-      state.live.isTeam ? 'Connected to Live Team!' : 'Connected to Live!'
-    );
+export const onJoin: Operator<
+  LiveMessage<{ status: 'connected'; live_user_id: string }>
+> = mutate(({ effects, state }, { data }) => {
+  state.live.liveUserId = data.live_user_id;
```

Comment:
IS this not when anyone joins? Or is it when only you join?

Reply:
Only when you join. good catch!